README.md

```diff
@@ -239,3 +239,13 @@ The [examples](https://github.com/grpc/grpc-java/tree/master/examples)
 and the
 [Android example](https://github.com/grpc/grpc-java/tree/master/examples/android) are standalone projects that
 showcase the usage of gRPC.
+
+Tools
+-----
+APIs that are annotated with `@ExperimentalApi` are subject to change, and APIs
```

Comment:
I think we want to aim this more at library code that uses gRPC, as they may have less control of the ultimate gRPC version that gets used than an application developer directly using gRPC. Something like:

> APIs that are annotated with `@ExperimentalApi` are subject to change, and APIs that are annotated with `@Internal` are for internal use by the gRPC library. Because APIs with these annotations could be modified or even removed in a future release, library code that other projects may depend on should not use these APIs. We recommend using the [grpc-java-api-checker](https://github.com/grpc/grpc-java-api-checker) (an [Error Prone](https://github.com/google/error-prone) plugin) to check for usages of `@ExperimentalApi` and `@Internal` in any library code that depends on gRPC.

Reply:
I thought all users should not use `@Internal` APIs. I respect your opinion :)

Reply:
That's fair. We have the wording in `@Internal`'s javadoc about using these APIs to implement a custom load balancer or transport, but that's probably fine to leave out here:

> APIs annotated with `@Internal` are for internal use by the gRPC library and should not be used by gRPC users. APIs annotated with `@ExperimentalApi` are subject to change in future releases, and library code that other projects may depend on should not use these APIs. We recommend using the [grpc-java-api-checker](https://github.com/grpc/grpc-java-api-checker) (an [Error Prone](https://github.com/google/error-prone) plugin) to check for usages of `@ExperimentalApi` and `@Internal` in any library code that depends on gRPC. It may also be used to check for `@Internal` usage or unintended `@ExperimentalApi` consumption in non-library code.

Currently quite a lot of our APIs are marked experimental, so I think the "you should not use `@ExperimentalApi`" statement in the original text is too strong. @ejona86 WDYT?

Reply:
Oops, the load balancer reference is out-dated. Sent out #4192

Yeah, "not use" is too strong. For the library piece I think I'd say something closer to "Since libraries can not generally guarantee which version of gRPC is being used, most libraries should not use APIs annotated with `@ExperimentalApi`." But @ericgribkoff's most recent suggested text looks fine.

Reply:
Thanks! I've fixed.